packages/back-end/src/tracing.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as opentelemetry from "@opentelemetry/sdk-node";
+import { diag, DiagConsoleLogger } from "@opentelemetry/api";
+import {
+  getNodeAutoInstrumentations,
+  getResourceDetectors,
+} from "@opentelemetry/auto-instrumentations-node";
+import { OTLPMetricExporter } from "@opentelemetry/exporter-metrics-otlp-proto";
+import { PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics";
+
+diag.setLogger(
+  new DiagConsoleLogger(),
+  opentelemetry.core.getEnv().OTEL_LOG_LEVEL
+);
+
+const metricReader = new PeriodicExportingMetricReader({
+  exporter: new OTLPMetricExporter(),
+  exportIntervalMillis: 1000,
+});
+
+const sdk = new opentelemetry.NodeSDK({
+  instrumentations: getNodeAutoInstrumentations(),
+  resourceDetectors: getResourceDetectors(),
```

Comment:
Looks like this is using `getResourceDetectorsFromEnv` in the built-in register script.  Was changing this intentional?

Reply:
[They're the same function](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/585b833dc8e911c17b483e871f752ad4988cee5a/metapackages/auto-instrumentations-node/src/index.ts#L19)